src/main/resources/META-INF/plugin.xml

```diff
@@ -92,14 +92,9 @@
     </description>
 
   <change-notes><![CDATA[
-    <h2>New Features</h2>
-      <ul>
-        <li>Replaced Activity Alert Dialog with a Notification Balloon which is non-blocking.</li>
-      </ul>
     <h2>Bug Fixes</h2>
       <ul>
-        <li>Fixed repeated activity alert being shown to some users.</li>
-        <li>Fixed populateBuffer issue for unopened synced project with new branch.</li>
+        <li>Fixed an error in handle buffer. Added checks for null values.</li>
```

Comment:
Looks great, I think we haven't updated in a long way, can we add recently deployed features as well?
Async Uploader
Account Reconnect
Disconnected Repo Handling
Pull Request Playback Integration
(Kindly add more if I missed anythin)